File: Python Folder/PRAK204-2210817310011-M.Daffa_Az-Zikra.py
pie = 3.14
r,t = input().split()
r = float(r)
t = float(t)
v = pie*r*r*t
lp = 2*pie*r*(r+t)
k = pie*(r+t)
v = format(v,".2f")
lp = format(lp,".2f")
k = format(k,".2f")
print('Volume =',v,'\nLuas =',lp,'\nKeliling =',k)

File: C Folder/PRAK204-2210817310011-M.Daffa_Az-Zikra.c
#include<stdio.h>
int main(){
    float r,t,pie,v,lp,k;
    pie = 3.14;
    scanf("%f %f",&r,&t);
    v = pie*r*r*t;
    lp = 2*pie*r*(r+t);
    k = pie*(r+r);
    printf("Volume = %.2f \nLuas = %.2f \nKeliling = %.2f",v,lp,k);
}